### ISSUE TITLE
Allow tracked coupons to be limited to certain products

### ISSUE THREAD

**Issue by sumobi** [affiliatewp/AffiliateWP#495]
Currently if a linked affiliate coupon is used at checkout the affiliate is awarded commission based on the entire purchase. If a coupon is limited to certain products this isn't honoured. Do you think this is something we should expand on?

From the customer:

<blockquote>
I tried to limit the coupon to a specific item in WooCommerce, but the entire order still is counted in the affiliate section.

In other words. If the the customer orders a hat and a bag on our site and received a coupon referral for the hat from our affiliate, we want to give them a referral disbursement for just the hat, not the hat and the bag. Is that possible?
</blockquote>

**Comment by pippinsplugins** [comment]
This will really depend on each e-commerce plugin.

WooCommerce does not subtract coupon amounts from line items.

EDD does subtract coupon amounts from line items.

We may be able to account for this but it will be tricky.

**Comment by sumobi** [comment]
:+1:
